app/code/community/Reverb/ReverbSync/Helper/Sync/Product.php

```diff
@@ -131,8 +131,9 @@ public function executeSimpleProductSync($simpleProduct, $do_not_allow_creation
         if ($productType == Mage_Catalog_Model_Product_Type::TYPE_SIMPLE)
         {
             //pass the data to create or update the product in Reverb
-            $listingWrapper = Mage::helper('ReverbSync/data')
-                                ->createOrUpdateReverbListing($simpleProduct, $do_not_allow_creation);
+            $reverbSyncHelper = Mage::helper('ReverbSync/data');
+            /* @var $reverbSyncHelper Reverb_ReverbSync_Helper_Data */
```

Comment:
Again, adding comments to denote the class of variable $reverbSyncHelper to make code traversal easier